inbm/dispatcher-agent/dispatcher/source/constants.py

```diff
@@ -25,16 +26,16 @@ class SourceParameters:
 
 @dataclass(kw_only=True, frozen=True)
 class ApplicationAddSourceParameters:
-    gpg_key_uri: str
-    gpg_key_name: str
     file_name: str
```

Comment:
**Suggestion:** Use a more descriptive variable name than file_name to indicate the purpose of the variable. [readability]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    @dataclass(kw_only=True, frozen=True)
    class ApplicationAddSourceParameters:
        source_list_file_name: str
```

Reply:
already changed.